Samples/Shared/CameraKsPropertyHelper/KSHelper.h

```diff
@@ -0,0 +1,146 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+#pragma once
+
+#include <ks.h>
+#include <ksmedia.h>
+
+namespace winrt::CameraKsPropertyHelper::implementation
+{
+
+    // Redefining structs normally available in ksmedia.h but not available due to WINAPI_PARTITION_APP
+    //

```

Comment:
Didn't you fix it in SV2? Perhaps we need ifdef of some kind?

Reply:
I did but seems missed few :(